### PR TITLE
web: add account address section on Home

### PR DESCRIPTION
**Pull request by maxime-carabina**
We are adding account address section on `Home` page.

### CODE REVIEW

File: web-frontend/src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -1,40 +1,63 @@
...
 export default function Home() {
+  const navigate = useNavigate();
   const { nickname } = useParams();
   const { data: account } = useResource<AccountObject>(`accounts/${nickname}`);
 
-  const unformattedBalance = account?.candidateBalance ?? '0';
+  if (account === undefined) {
+    navigate(routeFor(`index`));
+    return null;
+  }
```
Comment (mazmassa) [inline web-frontend/src/pages/Home/Home.tsx:22]: We are relying in the wrong hook. This is not the way. I'll show you just after the daily.

File: web-frontend/src/pages/Home/Home.tsx [hunk 2]
```diff
@@ -1,40 +1,63 @@
...
+            >
               {Intl.t('home.buttons.send')}
             </Button>
           </div>
         </div>
+        <div className="bg-secondary rounded-2xl w-full max-w-lg p-10">
+          <p className="mas-body text-f-primary mb-6">
+            {Intl.t('home.title-account-address')}
+          </p>
+          {/* <CopyContent content={address} formattedContent={formattedAddress} /> */}
```
Comment (mazmassa) [inline web-frontend/src/pages/Home/Home.tsx:58]: 🤔

Comment (maxime-carabina) [reply web-frontend/src/pages/Home/Home.tsx]: @mazmassa Corrected ! 👍🏻

File: web-frontend/src/pages/Home/Home.tsx (hunk 1 above)
Comment (maxime-carabina) [reply web-frontend/src/pages/Home/Home.tsx:22]: Yes! Let we sync to solve this problem.

**Pull request merged**